### PR TITLE
update node version to 14.0.0

### PR DESCRIPTION
**Pull request by goodviber**
### Context
Move to ruby 2.7.2 alpine etc.
### Changes proposed in this pull request
Update to ruby 2.7.2 and node 14.16.0
### Guidance to review
See if it will deploy

### CODE REVIEW

**Comment by ethanmills** [comment]
Snyk failure: per https://github.com/alpinelinux/docker-alpine/issues/123#issuecomment-776706144, since ruby:2.7.2-alpine uses musl-1.2.2-r0, I believe it is no longer vulnerable. This seems to be an issue parsing their weird semver on Snyk's end. This check is not required for merging in any case, and I'm happy to go ahead like this

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-12.18.3
+14.0.0
```
Comment (ethanmills) [inline .nvmrc:1]: Let's have 14.16.0 please

Comment (goodviber) [reply .nvmrc]: Done

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -12,13 +12,13 @@ jobs:
...
         with:
-          node-version: '12'
+          node-version: '14.x'
       - name: Yarn install
         run: npm i -g yarn && yarn
       - name: Set up Ruby
         uses: ruby/setup-ruby@9bc07a3af02ce5950e977cab8b77210dc3178328
         with:
-          ruby-version: 2.7.1
+          ruby-version: 2.7.2
```
Comment (kicferk1) [inline .github/workflows/lint.yml:21]: cheeky ruby version update too I see ;)

**Review by kicferk1 — APPROVED** [review]
LGTM, not sure what happened in git history though. Maybe interactive rebase on `develop` to purge old commits is a good idea?

**Comment by kicferk1** [comment]
Interactive rebase to blow away old commits will probably also help with the 'all commits must be signed' requirement.

File: .github/workflows/lint.yml (hunk 2 above)
Comment (goodviber) [reply .github/workflows/lint.yml:21]: The ruby version required the node update, I didn't phrase the description very well. I ran a branch filter post commit as I wasn't aware we had to sign our commits

**Pull request closed without merging**